frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -511,46 +525,181 @@ function ReaderFooter:addToMainMenu(menu_items)
-                checked_func = function()
-                    return self.settings.align == "left"
-                end,
-                callback = function()
-                    self.settings.align = "left"
-                    self:updateFooterContainer()
-                    self:resetLayout(true)
-                    self:updateFooter()
-                    UIManager:setDirty(nil, "ui")
-                end,
+                text = _("Text separator"),
+                sub_item_table = {
+                    {
+                        text = _("Vertical line") .. " (|)",
+                        checked_func = function()
+                            return self.settings.text_separator == "vertiacal_line" or self.settings.text_separator == nil
+                        end,
+                        callback = function()
+                            self.settings.text_separator = "vertiacal_line"
+                            self:updateFooter()
+                            UIManager:setDirty(nil, "ui")
+                        end,
+                    },
+                    {
+                        text = _("Dot") .. " (•)",
+                        checked_func = function()
+                            return self.settings.text_separator == "dot"
+                        end,
+                        callback = function()
+                            self.settings.text_separator = "dot"
+                            self:updateFooter()
+                            UIManager:setDirty(nil, "ui")
+                        end,
+                    },
+                    {
+                        text = _("Without separator"),
+                        checked_func = function()
+                            return self.settings.text_separator == "no_separator"
+                        end,
+                        callback = function()
+                            self.settings.text_separator = "no_separator"
+                            self:updateFooter()
+                            UIManager:setDirty(nil, "ui")
+                        end,
+                    },
+                },
             },
             {
-                text = _("Right"),
-                checked_func = function()
-                    return self.settings.align == "right"
-                end,
-                callback = function()
-                    self.settings.align = "right"
-                    self:updateFooterContainer()
-                    self:resetLayout(true)
-                    self:updateFooter()
-                    UIManager:setDirty(nil, "ui")
-                end,
+                text = _("Progress percentage format"),
+                sub_item_table = {
+                    {
+                        text = T(_("No decimal point (%1)"), self:progressPercentage(0)),
+                        checked_func = function()
+                            return self.settings.progress_perc_format == "0" or self.settings.progress_perc_format == nil
```

Comment:
`progress_perc_format ` => `progress_pct_format` ? "pct" feels more natural than "perc" to me.
Or progress_pct_digits to be more explicite. (What's the word for digit *after* the dot? fractional digits?)

Reply:
Are you talking about decimals? :-P

Reply:
Done.